src/app/participant-list/participant-list.component.ts

```diff
@@ -524,11 +525,12 @@ export class ParticipantListComponent implements OnInit {
         }
         if (jsonData.hideSamplesTab === true) {
           this.hideSamplesTab = true;
+          this.dataSources.delete( "k" );
```

Comment:
I hope one day we will eliminate all this hard coding things from frontend and backend!